examples/forms/src/tests.rs

```diff
@@ -35,6 +35,11 @@ fn test_bad_login() {
     test_login("Sergio", "jk", -100, OK, Some("'-100' is not a valid integer."));
     test_login("Sergio", "ok", 30, OK, Some("Wrong password!"));
     test_login("Mike", "password", 30, OK, Some("Unrecognized user, 'Mike'."));
+    // request parameters are url-encoded (normally by browser)
+    test_login("print_passsword", "password", 100, OK, Some("password"));
```

Comment:
These tests are valuable, but I think we should do them outside of the example. The best place for them, I think, in `lib/tests`. You can simple have a single route that returns the `String` directly for comparison.